app/src/main/java/org/mozilla/fenix/compose/SectionHeader.kt

```diff
@@ -20,12 +21,13 @@ import org.mozilla.fenix.theme.FirefoxTheme
 @Composable
 fun SectionHeader(
     text: String,
-    modifier: Modifier = Modifier
+    modifier: Modifier = Modifier,
+    textColor: Color = FirefoxTheme.colors.textPrimary,
```

Comment:
```suggestion
    color: Color = FirefoxTheme.colors.textPrimary,
```

Reply:
I prefer to have color usage named explicitly in parameters, to avoid renaming when adding colors to other elements, like background. Moreover, the naming is also used in the composables that pass this param down to the `SectionHeader`.

Reply:
One reason I actually prefer `color` here is that the component only wraps a `Text` and would mirror the parameters expected in `Text.`